static/app/components/dropdownMenuControlV2.tsx

```diff
@@ -121,6 +121,14 @@ function MenuControl({
     ref
   );
 
+  // Calculate the current trigger element's width. This will be used as
+  // the min width for the menu.
+  const [triggerWidth, setTriggerWidth] = useState<number>();
+  useEffect(() => {
+    const newTriggerWidth = ref.current?.offsetWidth;
+    !isSubmenu && newTriggerWidth && setTriggerWidth(newTriggerWidth);
```

Comment:
Adding `!isSubmenu` here since we don't need to make submenus span the width of their trigger elements.

<img width="389" alt="Screen Shot 2022-02-24 at 7 26 31 PM" src="https://user-images.githubusercontent.com/44172267/155648116-414a8e92-6d32-468a-a4c8-534d3d850908.png">